CHANGELOG.MD

```diff
@@ -0,0 +1,214 @@
+## 4.0
+- Added the ability to terminate a given application on the device (#340). See [documentation](http://appium.io/docs/en/commands/device/app/terminate-app)
+- Added the ability to query the given app status on a device (#341). See [documentation](http://appium.io/docs/en/commands/device/app/app-state/)
```

Comment:
a device or the device?

Reply:
Copy and paste from appium.io. Just a sec